examples/prometheus/prometheus.yaml

```diff
@@ -118,6 +125,7 @@ objects:
           - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
           - -cookie-secret-file=/etc/proxy/secrets/session_secret
           - -skip-auth-regex=^/metrics
+          - '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get", "name": "${NAMESPACE}"}}'
```

Comment:
Tested with v1. Works.
P.S there was a typo in oath-proxy documentation of this flag.
fixed in https://github.com/openshift/oauth-proxy/pull/4